### PR TITLE
Add scripts for migrating all sms / ivr backends and backend mappings

### PR DESCRIPTION
**Pull request by gcapalbo**
@biyeun

### CODE REVIEW

File: corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from corehq.apps.sms.mixin import BackendMapping
```
Comment (biyeun) [inline corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py]: @gcapalbo assuming you are running this after the next deploy?

Comment (gcapalbo) [inline corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py]: @biyeun yeah i was probably going to run it tomorrow at some point, but there's no particular urgency since all the code still is working off of the couch models. why do you ask?

Comment (gcapalbo) [inline corehq/apps/sms/management/commands/migrate_backend_mappings_to_sql.py]: (not sure if i made this clear but it's ok to merge this at any time and there's no need to coordinate with a deploy - these are management commands that i'll be running manually)

**Comment by czue** [comment]
are these migrations idempotent?

File: corehq/apps/sms/models.py [hunk 2]
```diff
@@ -1508,7 +1508,6 @@ class SQLMobileBackend(SyncSQLToCouchMixin, models.Model):
 
     class Meta:
         db_table = 'messaging_mobilebackend'
-        unique_together = ('domain', 'name')
```
Comment (czue) [inline corehq/apps/sms/models.py]: why was this necessary?

**Comment by czue** [comment]
(no blockers from me will merge after you respond to questions)

**Comment by gcapalbo** [comment]
@czue yes, they're idempotent

File: corehq/apps/sms/models.py (hunk 2 above)
Comment (gcapalbo) [inline corehq/apps/sms/models.py]: i realized that you can (soft) delete a backend and then create a new one with the same name (which is fine to do), but then this constraint fails.  so since the unique name per domain is enforced in the ui that should be enough for this

**Comment by czue** [comment]
:+1: thanks
